README.md

```diff
@@ -1,12 +1,27 @@
-# pie_of_the_month
-Google Chrome Extension to show what the Pinthouse pizza of the month currently is. For more information about Pinthouse pizza, please visit http://pinthousepizza.com/burnet/about.
+# Pie Of The Month
 
-File templates and info used to write this .md file from https://gist.github.com/PurpleBooth/109311bb0361f32d87a2#file-readme-template-md and https://github.com/gorhill/uBlock.
+> Google Chrome Extension to show the current Pinthouse pizza of the month.
 
 ## Prerequisites
+
 This extension was tested during development in Google Chrome. Run this in Chrome for best results.
 
+## Installation
+
+1. Clone this repo (extract if needed)
+2. Go to chrome extensions [chrome://extensions](chrome://extensions)
+3. Enable developer mode
+4. Click on load unpacked extension and select the extension folder.
+
+For more information see the Chrome Developer getting started [Tutorial](https://developer.chrome.com/extensions/getstarted)
+
```

Comment:
I wonder if we should break these installation instructions into two sections, one for the case when the user clones in the Git command line way and another for the case where the user just downloads the zip file? Some of my peeps will possibly not be able to use these simplified steps for both cases. 💯

Reply:
I agree the installation steps may differ, but only slightly, since cloning copies the project into a new folder.
I can see having a development set of instructions, and one for the end-user. However, specially because the in either scenario the user would have to turn on developer mode, it could be reduced to a sub-step or note.
Ideally the end user instructions would be a simple link to the chrome store.

Let me know!

Reply:
I can mark this as Resolved conversation once we resolve issue #10 !